### PR TITLE
improvement of the logging

### PR DESCRIPTION
**Pull request by kinnou02**

### CODE REVIEW

File: source/jormungandr/jormungandr/instance.py [hunk 1]
```diff
@@ -250,7 +251,8 @@ def send_and_receive(self,
                 socket.setsockopt(zmq.LINGER, 0)
                 socket.close()
                 if not quiet:
-                    logging.error("request: " + str(request) + " failed " + self.socket_path)
+                    logger = logging.getLogger(__name__)
+                    logger.error("request: " + str(request) + " failed " + self.socket_path)
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/instance.py]: use %s no ?

File: source/jormungandr/jormungandr/exceptions.py [hunk 2]
```diff
@@ -110,15 +111,13 @@ def __init__(self, msg):
...
     error = exception.__class__.__name__ + " " + message + " " + request.url
 
-    if hasattr(exception, "data"):
-        # if an exception is thrown with 'brief_log', we just log it in one line in debug
-        if 'brief_log' in exception.data:
-            logging.debug(error)
-            return
-
-    logging.exception(error)
+    if isinstance(exception, (HTTPException, RegionNotFound)):
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/exceptions.py:120]: I think we should keep the brief log facility

Comment (kinnou02) [inline source/jormungandr/jormungandr/exceptions.py:120]: And I don't think so,  we will need a little help here

Comment (TeXitoi) [inline source/jormungandr/jormungandr/exceptions.py:120]: I vote "he's blu^W^Wremove brief log"

**Pull request merged**